src/shared/mount-util.c

```diff
@@ -44,19 +44,22 @@
 #include "user-util.h"
 
 int umount_recursive(const char *prefix, int flags) {
+        _cleanup_fclose_ FILE *f = NULL;
```

Comment:
maybe a bit more descriptive name would be nicer, like `proc` or `fproc` or so

Reply:
(I do not think it is not necessary to rename. Only file '/proc/self/mountinfo' is opened in this function and `f` is well used name for `FILE*`.)